doc/api/crypto.md

```diff
@@ -1284,6 +1284,22 @@ passing keys as strings or `Buffer`s due to improved security features.
 The receiver obtains a cloned `KeyObject`, and the `KeyObject` does not need to
 be listed in the `transferList` argument.
 
+### `keyObject.asymmetricKeyDetails`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object}
+  * `modulusLength`: {number} Key size in bits (RSA, DSA).
+  * `publicExponent`: {bigint} Public exponent (RSA).
+  * `divisorLength`: {number} Size of `q` in bits (DSA).
+  * `namedCurve`: {string} Name of the curve (EC).
+
+This property exists only on asymmetric keys. Depending on the type of the key,
+this object contains information about the key. None of the information obtained
+through this property can be used to uniquely identify a key or to compromise
+the security of the key.
```

Comment:
Suggestion, feel free to ignore: Could you maybe add a note that PSS parameters and DH key details are not currently exposed through this API, but might be added in the future? Not sure if this is the best place.

Reply:
@tniessen In the future those would be added in a non-breaking fashion. Shall I add

> RSA-PSS parameters, DH, or any future key type details might be exposed via this API using additional attributes.

Reply:
SGTM!